htdocs/core/lib/prelevement.lib.php

```diff
@@ -50,7 +51,7 @@ function prelevement_prepare_head(BonPrelevement $object)
 	$h++;
 
 	$head[$h][0] = DOL_URL_ROOT.'/compta/prelevement/factures.php?id='.$object->id;
-	$head[$h][1] = $langs->trans("Bills");
+	$head[$h][1] = ($salary <= 0 ? $langs->trans("Bills") : $langs->trans("Salaries"));
```

Comment:
Kowning if this is a BonPrelevement for salary of invoice can be know with 
$object->checkIfSalaryBonPrelevement();